src/Okta.Sdk.UnitTests/UsersClientShould.cs

```diff
@@ -124,5 +125,464 @@ public async Task RemoveApplicationTargetFromApplicationAdministratorRoleForUser
+                                        ""displayName"": ""Drive car"",
+                                        ""description"": ""Allows the user to drive a car."",
+                                        ""_links"": {
+                                          ""scope"": {
+                                            ""href"": ""https://${yourOktaDomain}/api/v1/authorizationServers/ausain6z9zIedDCxB0h7/scopes/scp142iq2J8IGRUCS0g4"",
+                                            ""title"": ""Drive car""
+                                          }
+                                        }
+                                      }
+                                    ]
+                                  },
+                                  ""_links"": {
+                                    ""app"": {
+                                      ""href"": ""https://${yourOktaDomain}/api/v1/apps/0oabskvc6442nkvQO0h7"",
+                                      ""title"": ""Native""
+                                    },
+                                    ""self"": {
+                                      ""href"": ""https://${yourOktaDomain}/api/v1/users/00u5t60iloOHN9pBi0h7/clients/0oabskvc6442nkvQO0h7/tokens/oar579Mcp7OUsNTlo0g3""
+                                    },
+                                    ""revoke"": {
+                                      ""href"": ""https://${yourOktaDomain}/api/v1/users/00u5t60iloOHN9pBi0h7/clients/0oabskvc6442nkvQO0h7/tokens/oar579Mcp7OUsNTlo0g3"",
+                                      ""hints"": {
+                                        ""allow"": [
+                                          ""DELETE""
+                                        ]
+                                      }
+                                    },
+                                    ""client"": {
+                                      ""href"": ""https://${yourOktaDomain}/oauth2/v1/clients/0oabskvc6442nkvQO0h7"",
+                                      ""title"": ""Example Client App""
+                                    },
+                                    ""user"": {
+                                      ""href"": ""https://${yourOktaDomain}/api/v1/users/00upcgi9dyWEOeCwM0g3"",
+                                      ""title"": ""Saml Jackson""
+                                    },
+                                    ""authorizationServer"": {
+                                      ""href"": ""https://${yourOktaDomain}/api/v1/authorizationServers/ausain6z9zIedDCxB0h7"",
+                                      ""title"": ""Example Authorization Server""
+                                    }
+                                  }
+                                }";
+            var mockRequestExecutor = new MockedStringRequestExecutor(rawResponse);
+            var client = new TestableOktaClient(mockRequestExecutor);
+            await client.Users.GetRefreshTokenForUserAndClientAsync("foo", "bar", "baz");
```

Comment:
In this case, since I believe we don't have other tests that verify that the token is response is being parsed correctly, I suggest testing that at least the main properties (exclude properties that start with `_`, i.e: `_embedded`) are being parsed correctly. Something like this:
```
var oauthRefreshToken = await client.Users.GetRefreshTokenForUserAndClientAsync("foo", "bar", "baz");
oauthRefreshToken.ClientId.Should().Be("0oabskvc6442nkvQO0h7");
....
```